### PR TITLE
Fix for give setting page #1998

### PR DESCRIPTION
**Pull request by raftaar1191**
Fix for give setting page data loss on page reload

## Description
PR for issues #1998 

## How Has This Been Tested?
See the video 
Video link http://screencast-o-matic.com/watch/cbQl3MI2uT

## Types of changes
New feature (non-breaking change which adds functionality)

## Checklist:
- [ ] My code is tested.
- [ ] My code follows the WordPress code style.
- [ ] My code follows has proper inline documentation.

### CODE REVIEW

File: assets/js/admin/admin-scripts.js [hunk 1]
```diff
@@ -9,6 +9,8 @@
  */
 
 jQuery.noConflict();
+// Provided access to golbal level
```
Comment (DevinWalker) [inline assets/js/admin/admin-scripts.js]: typo here "global"

File: assets/js/admin/admin-scripts.js [hunk 2]
```diff
@@ -9,6 +9,8 @@
...
+var givesettingedit = false;
```
Comment (DevinWalker) [inline assets/js/admin/admin-scripts.js]: Please make it more legible with underscores.

File: assets/js/admin/admin-scripts.js (hunk 1 above)
Comment (raftaar1191) [reply assets/js/admin/admin-scripts.js]: Done 

Thanks @DevinWalker

File: assets/js/admin/admin-scripts.js (hunk 2 above)
Comment (raftaar1191) [reply assets/js/admin/admin-scripts.js]: Done

**Pull request merged**